tests/FSharp.Compiler.ComponentTests/CompilerOptions/fsc/misc/compiler_help_output.bsl

```diff
@@ -47,41 +47,41 @@ Copyright (c) Microsoft Corporation. All Rights Reserved.
 
 
                 - CODE GENERATION -
---debug[+|-]                             Emit debug information (Short form: -g)
+--debug[+|-]                             Emit debug information (Short form: -g) (off by default)
 --debug:{full|pdbonly|portable|embedded} Specify debugging type: full, portable, embedded, pdbonly. ('full' is the
                                          default if no debugging type specified and enables attaching a debugger to a
                                          running program, 'portable' is a cross-platform format, 'embedded' is a
                                          cross-platform format embedded into the output file).
---embed[+|-]                             Embed all source files in the portable PDB file
+--embed[+|-]                             Embed all source files in the portable PDB file (off by default)
 --embed:<file;...>                       Embed specific source files in the portable PDB file
 --sourcelink:<file>                      Source link information file to embed in the portable PDB file
---optimize[+|-]                          Enable optimizations (Short form: -O)
---tailcalls[+|-]                         Enable or disable tailcalls
+--optimize[+|-]                          Enable optimizations (Short form: -O) (on by default)
+--tailcalls[+|-]                         Enable or disable tailcalls (on by default)
 --deterministic[+|-]                     Produce a deterministic assembly (including module version GUID and
-                                         timestamp)
---realsig[+|-]                           Generate assembly with IL visibility that matches the source code visibility
+                                         timestamp) (off by default)
+--realsig[+|-]                           Generate assembly with IL visibility that matches the source code visibility (off by default)
 --pathmap:<path=sourcePath;...>          Maps physical paths to source path names output by the compiler
---crossoptimize[+|-]                     Enable or disable cross-module optimizations
+--crossoptimize[+|-]                     Enable or disable cross-module optimizations (on by default)
 --reflectionfree                         Disable implicit generation of constructs using reflection
 
 
                 - ERRORS AND WARNINGS -
---warnaserror[+|-]                       Report all warnings as errors
---warnaserror[+|-]:<warn;...>            Report specific warnings as errors
+--warnaserror[+|-]                       Report all warnings as errors (off by default)
+--warnaserror[+|-]:<warn;...>            Report specific warnings as errors (off by default)
```

Comment:
The default is an empty delimited list.
Plus means to report warnings as errors.
Minus should mean to exclude from the global (just above) warn as error feature.